src/registrar/views/health.py

```diff
@@ -2,4 +2,6 @@
 
 
 def health(request):
-    return HttpResponse("OK")
+    return HttpResponse(
+        '<html lang="en"><head><title>OK - Get.gov</title></head><body>OK</body>'
+    )
```

Comment:
At some future-ish point, I'd like to expand this health check to include the database.